website/docs/d/event_trigger.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: event_trigger"
+sidebar_current: "docs-mongodbatlas-datasource-event-trigger"
+description: |-
+    Describes an Event Trigger.
+---
+
+# mongodbatlas_event_trigger
+
+`mongodbatlas_event_trigger` describe an Event Trigger. 
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_event_trigger" "test" {
+  project_id = "PROJECT ID"
+  app_id = "APPLICATION ID"
+  name = "NAME OF THE TRINGGER"
+  type = "DATABASE"
+  function_id = "1"
+  disabled = false
+  config_operation_types = ["INSERT", "UPDATE"]
+  config_operation_type = "LOGIN"
```

Comment:
Two different types of triggers mixed here too, this is for an authentication trigger